act-rules-format.bs

```diff
@@ -15,86 +15,112 @@ Markup Shorthands: markdown yes
+Because ACT Rules rarely test the entirety of an accessibility requirement, passing ACT rules does not necessarily mean that an accessibility requirement is met. It is important to understand that **ACT Rules test non-conformance** to accessibility requirements. In some cases conformance can be inferred from the absence of failures, unlike WCAG sufficient techniques, ACT Rules should not be used for conformance claims unless the rule explicitly states it can be used that way. See [Rule Aggregation](#output-aggregation) for details.
 
 
-ACT Rule Structure {#structure}
-==============================
+ACT Rule Types {#rule-types}
+============================
 
-Rule Outline {#structure-outline}
------------------------
+In accessibility, there are often different technical solutions to make the same type of content accessible. These solutions could be tested in a single rule; however, such a rule tends to be quite complex, making it difficult to understand and maintain. The ACT Rules Format solves this by providing two types of rules:
 
-An ACT Rule MUST be written in plain language. It MUST consist of the following items:
+- [Atomic rules](#atomic-rule-outline) describe how to test a specific solution. It contains a precise definition of what elements, nodes or other parts of a web page are to be tested, and when those elements are considered to fail the rule. These rules should be kept small and atomic. Meaning that atomic rules test a single "pass condition", and do so without using results from other rules.
 
-* Descriptive title
-* [Unique identifier](#unique-identifier)
-* [Rule Description](#structure-description)
-* [Accessibility Requirements](#structure-accessibility-requirements)
-* [Limitations, Assumptions or Exceptions](#structure-limitations-assumptions-exceptions), if any
-* [Accessibility Support](#acc-support) (optional)
-* [Aspects Under Test](#input-aspects)
-* [Test Definition](#test-def)
+- [Composed rules](#composed-rule-outline) describe how results from atomic rules should be used to decide if a an accessibility requirement was failed. If there are multiple failures necessary for an accessibility requirement to fail, each "type" of failure should be written as separate atomic rules and be aggregated by a composed rule.
 
-Unique Identifier {#unique-identifier}
-----------------------------------------
+The separation between atomic rules and composed rules creates a division of responsibility. Atomic rules test if web content correctly implemented a particular solution. Composed rules test if a combination of `pass` outcomes from atomic rules is sufficient for an accessibility requirement not to `fail`. Not all atomic rules have to be part of a composed rule. Only if failing that atomic rule does not directly fail an [Accessibility Requirements](#structure-accessibility-requirements). An atomic rule MAY be part of multiple composed rules.
```

Comment:
"Only if failing that atomic rule does not directly fail an [Accessibility Requirements](#structure-accessibility-requirements)." needs to be better tied into the surrounding text, e.g.: 
"Atomic rules only have to be part of a composed rule if failing that atomic rule does not directly fail an [Accessibility Requirements](#structure-accessibility-requirements)."

Reply:
minor edit to 2nd sentence: "...if web content is correctly implemented in a particular solution."
minor edit to 2nd to last sentence: Remove the "an" to read "...does not directly fail Accessibility Requirements."  Or if you want the "an", remove the "s" in "Requirements".